src/main/java/core/basesyntax/service/impl/FruitTransactionProcessor.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.impl.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.TransactionProcessor;
+
+public class FruitTransactionProcessor implements TransactionProcessor<FruitTransaction> {
+    private OperationStrategy<FruitTransaction> strategy;
+
+    public FruitTransactionProcessor(OperationStrategy<FruitTransaction> strategy) {
+        this.strategy = strategy;
+    }
+
+    @Override
+    public void process(FruitTransaction transaction) {
+        OperationHandler<FruitTransaction> operationStrategy = strategy.get(transaction);

```

Comment:
this is a variable of type `OperationHandler` it should not be named `operationStrategy`